server/service/apple_mdm.go

```diff
@@ -613,6 +619,25 @@ func (svc *Service) GetMDMAppleConfigProfile(ctx context.Context, profileUUID st
 	return cp, nil
 }
 
+func (svc *Service) GetMDMAppleDeclaration(ctx context.Context, profileUUID string) (*fleet.MDMAppleDeclaration, error) {
```

Comment:
We could probably find a way handle the service layer without three different functions for the get endpoint. For now, we're duplicating a bit of code.